app/enterprise/2.1.x/kong-manager/oidc-mapping.md

```diff
@@ -0,0 +1,258 @@
+---
+title: OIDC Authenticated Group Mapping
+---
+## Introduction
+
+Using Kong's OpenID Connect plugin (OIDC), you can map identity provider (IdP)
+groups to Kong roles. Adding a user to Kong in this way gives them access to
+Kong based on their group in the IdP.
+
+After starting Kong Enterprise with the desired configuration, you can
+create new admins whose usernames match those in your IdP. Those
+users will then be able to accept invitations to join Kong Manager and log in
+with their IdP credentials.
+
+If an admin's group changes in the IdP, their Kong admin account's associated
+role also changes in Kong Enterprise the next time they log in through Kong
+Manager. The mapping removes the task of manually managing access in Kong
+Enterprise, as it makes the IdP the system of record.
+
+Here's how OIDC authenticaticated group mapping works:
+1. Create roles in Kong Enterprise using either the Kong Admin API or Kong
+Manager.
+2. Create groups and associate roles with the groups.
+3. Configure the OIDC plugin to connect with your IdP.
+4. When users log in to Kong Manager, they get permissions based on the IdP
+ group(s) they belong to.
+
+## Prerequisites
```

Comment:
Should probably add "Sessions plugin configured" - this is something we include as part of the install instructions, but likely worth a mention here.